app/views/menu/menu.html.erb

```diff
@@ -88,13 +88,46 @@
           </script>
           <% end %>
           <table class="table table-striped">
+          <tr>
+    <th>Dish Name</th>
+    <th>Dinner/Supper</th>
```

Comment:
The dinner/supper stars should be inside the if statement too. To average Mayflower residents, this is extraneous unnecessary information that won't make sense to them.

Reply:
Make sense, this is a easy change.